projects/packen-ng/README.md

```diff
@@ -648,6 +648,7 @@ A special input to show more details.
 - **description**: Set a smaller description text side label.
 - **isInput**: Defines whether to show input when clicked.
 - **mask**: Input masks for external library [ngx-mask](https://www.npmjs.com/package/ngx-mask)
+- **isPlaceholder**: Defines if the title show styles placeholder **NOTE** Only is used when component is child.
```

Comment:
Defines if the title **shows** style placeholder.  Only used when the component is a child.